.github/workflows/main.yml

```diff
@@ -150,7 +150,7 @@ jobs:
           branch="${GITHUB_REF##*/}"
           sed -i "s/VERSION_PLACEHOLDER/$branch/g" metadata.yaml
           cd OctoBot
-          python start.py tentacles -m "../metadata.yaml" -d "../new_tentacles" -p "../../any_platform.zip" -upe ${{ secrets.NEXUS_DEV_PATH }}/packages/full/${{ secrets.TENTACLES_REPOSITORY_NAME }}/${branch////_}
+          python start.py tentacles -m "../metadata.yaml" -d "../new_tentacles" -p "../../any_platform.zip" -upe ${{ secrets.NEXUS_DEV_PATH }}/packages/full/${{ secrets.TENTACLES_REPOSITORY_NAME }}
```

Comment:
no more branch ?

Reply:
It's automatically set from metadata.yaml. It uses `$NAME/any_platform.zip`.